src/main/java/xyz/nucleoid/plasmid/command/GameCommand.java

```diff
@@ -235,12 +258,74 @@ private static int listGames(CommandContext<ServerCommandSource> context) {
         return Command.SINGLE_SUCCESS;
     }
 
-    private static SuggestionProvider<ServerCommandSource> gameSuggestions() {
-        return (ctx, builder) -> {
-            return CommandSource.suggestMatching(
-                    GameConfigs.getKeys().stream().map(Identifier::toString),
-                    builder
-            );
-        };
+    private static int openChannel(CommandContext<ServerCommandSource> context) throws CommandSyntaxException {
```

Comment:
`context.getSource()` is called twice in here, so perhaps it'd be better to have it be a local variable?
This nitpick also applies for `removeChannel()` and `connectBlockToChannel()`